test/unit/specs/components/common/TmBalance.spec.js

```diff
@@ -4,10 +4,10 @@ import TmBalance from "common/TmBalance"
 
 describe(`TmBalance`, () => {
   let wrapper
-  let { mount } = setup()
+  let { shallow } = setup()
 
   beforeEach(() => {
-    let instance = mount(TmBalance, {
+    let instance = shallow(TmBalance, {
```

Comment:
why was this changed?

Reply:
Example: If you use a property `address` for `ShortAdress` in `TmBalance` but the actually property used in `ShortAdress` is `bech32` the `TmBalance` test will pass even though the actual application will fail.